### ISSUE TITLE
Erroneous "multiple accounts" warning. 

### ISSUE THREAD

**Issue by rewolff** [owncloud/client#9951]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

After upgrading my workstation, getting owncloud working again was a bit troublesome. Today I noticed a directory not being synced and I started owncloud. Possibly the first time that it did not "segv" upon startup. 

Now for 2 out of 3 synced directories it reports "multiple accounts are sharing the folder .... this configuration is known to lead to dataloss and is no longer supported". 

This is not true. I have my own Owncloud server and only one account exists on the sever. 

### Expected behavior

it would just normally sync without big RED lies. (the statement in red is FALSE, therefore it is a lie). 

### Steps to reproduce the issue

What exactly causes this: I don't know.

I've found two threads from september 2021 indicating that an upgrade to 2.9 caused this. I'm at 2.10. 
Those threads hint at something about hidden files. Those are internal to owncloud: I don't mess with them. 

### Screenshots

I have prepared a screenshot, I'm not a drag-and-drop person. I can upload a file by selecting it (BROWSE). 

### Logs

I don't know where "LOGs" are. 

### Client version number

Owncloud version 2.10.1 build 7389. 
(it doesn't mention anything about libraries). 

### Desktop environment (Linux only)

Ubuntu 22.04 LTS

### Client package version and origin (Linux only)

/var/lib/apt/lists/download.owncloud.com_desktop_ownCloud_stable_latest_linux_Ubuntu%5f22.04_Packages

### Server information

Please explain in the "help text" how to find it and I'd include it.

**Comment by TheOneRing** [comment]
https://doc.owncloud.com/desktop/next/faq.html#multiple-accounts-sharing-the-folder

**Comment by rewolff** [comment]
So owncloud managed to leave internal files behind and then blames it on me. Something I did in January and/or december caused this. Good luck finding the cause of THAT!

Tip: Make the error message read something like: "An other .sync file was found in your local directory. This usually means that you might also be sharing this through another account. See <URL> for more info".

Note that after reading the document you referred to I started looking for the sync files on the owncloud server. Probably obvious to you developers, but not to a "random user" like me. 

 In general: If you make the error message state the source of your conclusion, this helps users solve the problem. An erorr message that just displays the "result" that you THINK might be the cause, then when that conclusion happens to be wrong you put people on the wrong path instead of helping them find a solution. 
 
 And for sure, you're rubbing ME the wrong way when you accuse me of using a weird configuration when that isn't the case. It doesn't give me ANY hint as to what's wrong and I can only do a google search for the error message. That search resulted in two threads that blamed it on an upgrade and not to the "official documentation" on this.
